get_alma_bib_records.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import tomllib
+import json
+from pymarc import Record
+from alma_api_client import AlmaAPIClient, get_pymarc_record_from_bib
+
+
+def _get_args() -> argparse.Namespace:
+    """Returns the command-line arguments for this program."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--config_file", help="Path to configuration file", required=True
+    )
+
+    parser.add_argument(
+        "--input_file",
+        help="Path to input JSON file containing MMS IDs",
+        required=True,
+    )
+    parser.add_argument(
+        "--output_file",
+        help="Path to MARC output file which will be written from report data",
+        required=True,
+    )
+    args = parser.parse_args()
+    return args
+
+
+def _get_config(config_file_name: str) -> dict:
+    """Returns configuration for this program, loaded from TOML file."""
+    with open(config_file_name, "rb") as f:
+        config = tomllib.load(f)
+    return config
+
+
+def get_deduped_mms_ids(data: list[dict]) -> list[str]:
+    """Returns a list of unique MMS IDs from a list of dicts."""
+    mms_ids = []
+    for item in data:
```

Comment:
This can be simplified.  Using a set provides automatic de-duplication, so just:
`mms_ids = set(item["MMS Id"] for item in data)`
I don't think there's a need to cast this set back to a list - the rest of the program just needs an iterable - so change return type as well.